common/coat_of_arms/coat_of_arms/00_wc_landed_titles.txt

```diff
@@ -279,4 +279,31 @@ d_burning_blade = {
 		color1 = "red"
```

Comment:
The CoA of the Holy Light is weird.
It's pixilated and has weird color (the standard blue color is 0, 0, 128 RGB)
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/121798087-50735200-cc35-11eb-81c3-64ff669d05b2.png)

</details>

Reply:
New one in cabd096e0